lib/did-io.js

```diff
@@ -128,7 +128,8 @@ api.use = function(name, injectable) {
         globalName = 'jsonldjs';
         break;
       case 'uuid':
-        requireName = 'node-uuid';
+        // NOTE: the `uuid` module does not work in the browser
```

Comment:
Right, the `uuid` module requires a build step to work in the browser.  So our standby uuid solution does not have a npm compatible implementation for the browser.

In one of the test frameworks I have used the guid() method from `chance` https://github.com/chancejs/chancejs, https://www.npmjs.com/package/chance

The minimized version of chance is about 164K which is too heavy to use just for that one function.

I guess this will need to be tabled until an appropriate uuid solution can be found.